packages/identity-app/src/Identity.tsx

```diff
@@ -1,20 +1,88 @@
-// Copyright 2017-2018 @polkadot/light-apps authors & contributors
+// Copyright 2017-2018 @paritytech/substrate-light-ui authors & contributors
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 import React from 'react';
-import { Container } from '@polkadot/ui-components';
+import { RouteComponentProps } from 'react-router';
+import { Container, WalletCard } from '@polkadot/ui-components';
+import Grid from 'semantic-ui-react/dist/commonjs/collections/Grid';
 
-type Props = {
-  basePath: string;
+type Props = RouteComponentProps & {
+  basePath: string
 };
 
-type State = {};
+type State = {
+  seed?: string,
+  name?: string,
+  lookupAddress?: string
+};
 
 export class Identity extends React.PureComponent<Props, State> {
   state: State = {};
 
+  _handleInputSeed = (value: string) => {
```

Comment:
_something is a javascript convention to denote private functions in a class. In TS, we actually have the private modifier, so here the _ is not necessary!